google/cloud/aiplatform/jobs.py

```diff
@@ -987,6 +1033,20 @@ def __init__(
             ),
         )
 
+    @property
+    def network(self) -> Optional[str]:
```

Comment:
Since this is on both CustomJob and HPTuning job can this be centralized to _RunnableJob?

Reply:
In the HPTuning job `network` is set inside of `gca_resource.trial_job_spec.network` whereas CustomJob stores it in `gca_resource.network`

Reply:
Got it.